tests/acstone-programs/000.main/Makefile

```diff
@@ -0,0 +1,18 @@
+CC		:=	riscv64-unknown-elf-gcc
+OBJDUMP := riscv64-unknown-elf-objdump --disassemble-all --disassemble-zeroes --section=.text --section=.text.startup --section=.data
+
+TARGET	:= 000.main
+GCC_OPTS = -m32 -Wa,-march=RV32IMA -std=gnu99 -msoft-float
+LINK_OPTS = -m32 -nostartfiles -lc -lm
+LIB_DIR	:=	-L../libac_sysc
```

Comment:
Why Makefile paths are relative to the "acstone-programs" folder when they're located in a subfolder?
Wouldn't it be better if the path here were "-L../../libac_sysc" (same thing with all relative paths in this file), so the user can just go to this folder and type "make" instead of copying this source to another folder?